src/devices/shinasystem.ts

```diff
@@ -1119,4 +1119,61 @@ export const definitions: DefinitionWithExtend[] = [
             await endpoint.read("ssIasZone", ["currentZoneSensitivityLevel"]);
         },
     },
+    {
+        zigbeeModel: ["TQM-300Z"],
+        model: "TQM-300ZB",
+        vendor: "ShinaSystem",
+        ota: true,
+        description: "SiHAS Round Temperature/Humidity Sensor",
+        extend: [m.temperature({reporting: {min: 30, max: 300, change: 30}}), m.humidity({reporting: {min: 30, max: 3600, change: 60}}), m.battery()],
+    },
+    {
+        zigbeeModel: ["WCM-300Z"],
+        model: "WCM-300Z",
+        vendor: "ShinaSystem",
+        ota: true,
+        description: "SiHAS 4-gang wall outlet",
+        extend: [
+            // Endpoint 1 is not registered as it represents the entire device.
+            m.deviceEndpoints({endpoints: {p1: 2, p2: 3, p3: 4, p4: 5}}),
+            m.onOff({endpointNames: ["p1", "p2", "p3", "p4"], powerOnBehavior: false}),
+            m.electricityMeter({endpointNames: ["p1", "p2", "p3", "p4"], cluster: "metering"}),
+            m.binary({
+                name: "button_lock_mode",
+                valueOn: ["Lock", 1],
+                valueOff: ["Unlock", 0],
+                cluster: "genOnOff",
+                attribute: {ID: 0x900e, type: 0x20},
+                description: "Enables/disables the physical input lock for Button 1.",
+                endpointName: "p1",
+            }),
+            m.binary({
+                name: "button_lock_mode",
+                valueOn: ["Lock", 1],
+                valueOff: ["Unlock", 0],
+                cluster: "genOnOff",
+                attribute: {ID: 0x900e, type: 0x20},
+                description: "Enables/disables the physical input lock for Button 2.",
+                endpointName: "p2",
+            }),
+            m.binary({
+                name: "button_lock_mode",
+                valueOn: ["Lock", 1],
+                valueOff: ["Unlock", 0],
+                cluster: "genOnOff",
+                attribute: {ID: 0x900e, type: 0x20},
+                description: "Enables/disables the physical input lock for Button 3.",
+                endpointName: "p3",
+            }),
+            m.binary({
+                name: "button_lock_mode",
+                valueOn: ["Lock", 1],
```

Comment:
```suggestion
                valueOn: ["lock", 1],
```

(same for the others)